### ISSUE TITLE
Is this different than the platform.device.uuid?

### ISSUE THREAD

**Issue by NathanaelA** [hamorphis/nativescript-ios-uuid#1]
Is this a different uuid than:

```
var platform = require('platform');
console.log('platform.device.uuid');
```

**Comment by NathanWalker** [comment]
@NathanaelA Looks like this is different. I was just investigating this and it looks like it stores a unique id in the iOS keychain on the device: http://stackoverflow.com/questions/21878560/how-to-preserve-identifierforvendor-in-ios-after-uninstalling-ios-app-on-device and uses the CFBundleName to restore the same UUID for the app if a user deletes and reinstalls the app: https://github.com/hamorphis/nativescript-ios-uuid/blob/master/index.ios.js#L2 ... 
@hamorphis please correct me if this no longer works... I'm about to try it.

**Comment by roblav96** [comment]
Does this require iCloud to be in your iOS app identifier?

**Comment by hamorphis** [comment]
@NathanaelA I had abandoned this plugin. I will try to update it today. See this issue: https://github.com/hamorphis/nativescript-ios-uuid/issues/3

**Comment by hamorphis** [comment]
@NathanaelA @NathanWalker @roblav96 I released v1.0.0: https://www.npmjs.com/package/nativescript-ios-uuid. Please, give it a try and let me know if something does not work properly.
